API/src/main.ts

```diff
@@ -1,8 +1,24 @@
 import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
+import { appConfig } from './config';
+
+//import './database'; //Init database connection
 
 async function bootstrap() {
     const app = await NestFactory.create(AppModule);
-    await app.listen(3000);
+
+    const options = new DocumentBuilder()
+        .setTitle('Project Lockdown API')
+        .setDescription('This the api of project lockdown')
+        .setVersion('2.0')
+        .addTag('ProjectLockdown')
+        .build();
+
+    const apiDocument = SwaggerModule.createDocument(app, options);
+    SwaggerModule.setup('/docs', app, apiDocument);
```

Comment:
I don't think we want to combine Swagger with Stoplight. I'm not a big fan of Stoplight, so we can switch, but not in this PR, and instead we'll need to start a discussion https://github.com/TheIOFoundation/ProjectLockdown/discussions

Reply:
okay, I'll do research about them a bit.